docs/slice-12-preflight-notes.md

```
@@ -0,0 +1,390 @@
+Emit INSIDE `app.post("/", ...)` at line 95, BEFORE the signature/timestamp
+header reads.
+
+EXACT call (literal string, no template):
+
+```ts
+console.log("hubspot-lifecycle-webhook: request received");
+```
+
+Sample line in Vercel logs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)